var/spack/repos/builtin/packages/py-cinema-lib/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCinemaLib(PythonPackage):
+    """cinema_lib is a set of tools and library for interacting with a
+    Cinema database (currently Spec A and Spec D) through Python and
+    the command line tool, cinema."""
+
+    homepage = "https://cinemascience.github.io/"
+    url = "https://github.com/cinemascience/cinema_lib/archive/1.2.1.tar.gz"
+    list_url = "https://github.com/cinemascience/cinema_lib/archive"
+    list_depth = 1
```

Comment:
`list_url` and `list_depth` aren't required here, Spack knows where to find new versions of GitHub packages, and these values happen to be the default anyway.

Reply:
Done.